pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -162,7 +162,7 @@ def get_metadata_schema(source_data: Dict[str, dict], interfaces: dict) -> Dict[
 
     # Delete Ecephys metadata if ElectrodeTable helper function is not available
```

Comment:
```suggestion
    # Delete Ecephys metadata if ElectrodeTable helper function is not available (temporary solution)
```

Just adding a note how this is temporary to begin with